examples/api/test/widgets/widget_state/widget_state_property.0_test.dart

```diff
@@ -0,0 +1,95 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/widget_state/widget_state_property.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  Future<Color?> getTextColor(WidgetTester tester) async {
+    final BuildContext context = tester.element(find.text('TextButton'));
+    final TextStyle textStyle = DefaultTextStyle.of(context).style;
+
+    return textStyle.color;
+  }
+
+  testWidgets('displays red colored text by default', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.WidgetStatePropertyExampleApp(),
+    );
+
+    expect(
+      await getTextColor(tester),
+      equals(Colors.red),
+    );
```

Comment:
```suggestion
    expect(getTextColor(tester), Colors.red);
```

Once the utility function will be simplified, consider updating all the expect to a shorter syntax such as the proposed one.